cli/src/action/circuit/payload.rs

```diff
@@ -71,21 +74,21 @@ where
     header.set_payload_sha512(hashed_bytes.to_vec());
     header.set_requester(public_key);
     header.set_requester_node_id(requester_node.into());
-    let header_bytes = header.write_to_bytes().map_err(|err| {
-        CliError::ActionError(format!("Unable to serialize header to bytes: {}", err))
-    })?;
+    let header_bytes = header
+        .write_to_bytes()
+        .map_err(|_| CliError::ActionError("Failed to serialize payload header".into()))?;
 
     let header_signature = signer
         .sign(&header_bytes)
-        .map_err(|err| CliError::ActionError(format!("Unable to sign payload header: {}", err)))?;
+        .map_err(|_| CliError::ActionError("Failed to sign payload header".into()))?;
```

Comment:
Since we have logging setup, should we log the error to `debug!` so it would show-up during verbose logging?

Reply:
Changed the commit so it doesn't remove the internal errors at the request of @vaporos